src/swap/components/SwapProvider.tsx

```diff
@@ -233,6 +236,7 @@ export function SwapProvider({
   ]);
 
   const value = useValue({
+    error,
```

Comment:
This makes me think so much we need to refactor `getErrorMessage`. And this `error` story.